runtime/runtime_test.go

```diff
@@ -174,15 +174,15 @@ hello_re = 'he[l]{2}o\\sthere'
 def main():
```

Comment:
This file should give you an idea of the types of breaking changes this introduces.